include/ccsp_hal_ethsw.h

```diff
@@ -540,7 +540,7 @@ INT CcspHalExtSw_getEthWanEnable(BOOLEAN *pFlag);
  * @retval RETURN_ERR - On failure.
  */
 INT CcspHalExtSw_setEthWanEnable(BOOLEAN Flag);
-
+#ifdef FEATURE_RDKB_AUTO_PORT_SWITCH
```

Comment:
interfaces should not have #ifdef switches in them, the interface is fixed for all platforms.

Reply:
We cannot change the code, can we add a TODO?

Reply:
yes do that.